vnext/Microsoft.ReactNative/Fabric/Composition/ActivityIndicatorComponentView.cpp

```diff
@@ -0,0 +1,176 @@
+    const winrt::Microsoft::ReactNative::Composition::ICompositionContext &compContext,
+    facebook::react::Tag tag,
+    winrt::Microsoft::ReactNative::ReactContext const &reactContext)
+    : Super(compContext, tag), m_context(reactContext) {
+  m_props = std::make_shared<facebook::react::ActivityIndicatorViewProps const>();
+}
+
+void ActivityIndicatorComponentView::mountChildComponentView(
+    IComponentView &childComponentView,
+    uint32_t index) noexcept {
+  assert(false);
+}
+
+void ActivityIndicatorComponentView::unmountChildComponentView(
+    IComponentView &childComponentView,
+    uint32_t index) noexcept {
+  assert(false);
+}
+
+void ActivityIndicatorComponentView::handleCommand(std::string const &commandName, folly::dynamic const &arg) noexcept {
+  Super::handleCommand(commandName, arg);
+}
+
+void ActivityIndicatorComponentView::updateProps(
+    facebook::react::Props::Shared const &props,
+    facebook::react::Props::Shared const &oldProps) noexcept {
+  const auto &oldViewProps = *std::static_pointer_cast<const facebook::react::ActivityIndicatorViewProps>(m_props);
+  const auto &newViewProps = *std::static_pointer_cast<const facebook::react::ActivityIndicatorViewProps>(props);
+
+  ensureVisual();
+
+  updateBorderProps(oldViewProps, newViewProps);
+  m_props = std::static_pointer_cast<facebook::react::ViewProps const>(props);
+}
+
+void ActivityIndicatorComponentView::updateState(
+    facebook::react::State::Shared const &state,
+    facebook::react::State::Shared const &oldState) noexcept {}
+
+void ActivityIndicatorComponentView::updateLayoutMetrics(
+    facebook::react::LayoutMetrics const &layoutMetrics,
+    facebook::react::LayoutMetrics const &oldLayoutMetrics) noexcept {
+  // Set Position & Size Properties
+  ensureVisual();
+
+  if ((layoutMetrics.displayType != m_layoutMetrics.displayType)) {
+    OuterVisual().IsVisible(layoutMetrics.displayType != facebook::react::DisplayType::None);
+  }
+
+  updateBorderLayoutMetrics(layoutMetrics, *m_props);
+  m_layoutMetrics = layoutMetrics;
+
+  UpdateCenterPropertySet();
+  m_visual.Size(
+      {layoutMetrics.frame.size.width * layoutMetrics.pointScaleFactor,
+       layoutMetrics.frame.size.height * layoutMetrics.pointScaleFactor});
+}
+
+void ActivityIndicatorComponentView::finalizeUpdates(RNComponentViewUpdateMask updateMask) noexcept {
+  ensureDrawingSurface();
+}
+
+void ActivityIndicatorComponentView::prepareForRecycle() noexcept {}
+
+facebook::react::Props::Shared ActivityIndicatorComponentView::props() noexcept {
+  return m_props;
+}
+
+void ActivityIndicatorComponentView::ensureVisual() noexcept {
+  if (!m_visual) {
+    m_visual = m_compContext.CreateSpriteVisual();
+    m_ActivityIndicatorVisual = m_compContext.CreateActivityVisual(); // creates COM control
+
+    OuterVisual().InsertAt(m_ActivityIndicatorVisual, 0);
+    OuterVisual().InsertAt(m_visual, 0);
+  }
+}
+
+void ActivityIndicatorComponentView::ensureDrawingSurface() noexcept {
```

Comment:
There is no custom rendering here, so you shouldn't use or create a drawingSurface